_data/backup.yml

```diff
@@ -150,11 +150,12 @@ websites:
     img: onedrive.png
     tfa:
       - sms
+      - email
       - proprietary
       - totp
       - u2f
     doc: https://support.microsoft.com/en-us/help/12408/
-    exception: "Hardware token supporting FIDO2/WebAuthn can only be used in Windows 10 version 1809 and above using Edge browser."
+    exception: "Hardware token supporting FIDO2/WebAuthn can only be used in Windows 10 version 1809 and above."
```

Comment:
Can this exception be removed entirely from the Microsoft services it is listed on? version 1809 came out nearly 2 years ago.

Reply:
Yeah I don't see why not 🤷‍♂️